docs/api-site-config.md

```diff
@@ -251,10 +251,22 @@ Path to your web app manifest (e.g., `manifest.json`). This will add a `<link>`
 
 Override default [Remarkable options](https://github.com/jonschlinkert/remarkable#options) that will be used to render markdown.
 
+> To manage [syntax extensions](https://github.com/jonschlinkert/remarkable#syntax-extensions), use the `markdownPlugins` field.
+
 #### `markdownPlugins` [array]
 
 An array of plugins to be loaded by Remarkable, the markdown parser and renderer used by Docusaurus. The plugin will receive a reference to the Remarkable instance, allowing custom parsing and rendering rules to be defined.
 
+For example, if you want to [enable superscript and subscript](https://github.com/jonschlinkert/remarkable#syntax-extensions) in your markdown that is rendered by Remarkable to HTML, you would do the following:
+
+```
+  markdownPlugins: [
```

Comment:
This does not work at all

Reply:
Hi @DianaKoenraadt -- can you be a bit more specific? Are you trying to create a new site or is this for an existing site? Are you using v1 or v2 of Docusaurus? Thanks.

Reply:
Hi Joel, thanks for your message. We're using a v1 docusaurus website. We can't use ^bla^ or ~bla~, it doesn't have any result, even if we add this. We've also tried to set html: true but `sup` and `sub` also don't work.

Reply:
Are you able to share the site you are trying to enable this for?

cc @yangshun @slorber

Reply:
Hi @JoelMarcey, this would be docs.jodibooks.com. The exact page is not on there, still in development.

Reply:
Hi @joeplaa, can you open a separate issue so that we track this bug please? 

Also, it would be better to have a standalone repro case of the issue, or at least having access to the source of your website. Browsing a page there the problem is not will not help to fix the bug ;)

Reply:
Hi @slorber,

Yeah I thought the question "Are you able to share the site you are trying to enable this for?" was a bit strange.... took that way too literally 🤦 

Also, didn't saw we initially replied to a PR... We'll open a new issue.